Gemfile

```diff
@@ -51,9 +51,9 @@ gem 'rails_12factor', group: :production
 gem 'devise'
 
 #include bootstrap
-gem 'bootstrap-sass', '~> 3.3.5.1'
-gem 'sprockets', '2.11.0'
+gem 'bootstrap-sass', '~> 3.3.6'
+gem 'sprockets', '>= 3.2'
 
-# OmniAuth Authentication 
+# OmniAuth Authentication
 gem 'omniauth-google-oauth2'
 gem 'omniauth-facebook'
```

Comment:
Looks like that's the reason. If you get a chance to do so, you could review and merge #5 to speed things up. Based on the comments left by @lasley, this branch will need RSpec tests before it can be merged, so that will kill two birds with one stone.